test/utils.js

```diff
@@ -0,0 +1,6 @@
+exports.userTest = {
```

Comment:
just user is fine :) 
being in test directory already told other people that is used for testing purposes

Reply:
The problem is that i'm using variables named as user, so i need a unique name :o

Reply:
you can use other name such as createdUser